tests/schemas/api_v3/dryrunTransaction.schema.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import Joi from 'joi';
+
+const regex = require('./regex');
+
+const event = {
+	data: Joi.string().object().required(),
+	index: Joi.number().integer().min(0).required(),
+	module: Joi.string().pattern(regex.MODULE).required(),
+	name: Joi.string().pattern(regex.NAME).required(),
+	topics: Joi.array().items(Joi.string().pattern(regex.TOPIC)).required(),
+	height: Joi.number().positive().required(),
```

Comment:
Please re-verify if the dry run events have the `height` property set? I believe not.